packages/automerge-repo-bundles/test/bundles.test.ts

```diff
@@ -112,4 +113,90 @@ describe("when exporting and importing bundles", () => {
     await findWithProgress.handle.whenReady(["ready"])
     assert.deepStrictEqual(findWithProgress.handle.doc(), { foo: "bar" })
   })
+
+  it("repo persists the bundle data to storage", async () => {
```

Comment:
The other tests follow the pattern that `it` starts a sentence asserting what's being tested. This should do the same thing. E.g.,
```suggestion
  it("persists the bundle data to storage", async () => {
```
Same below.